roles/workload-agent/tasks/main.yml

```diff
@@ -17,6 +17,32 @@
   include_tasks:
     file: install.yml
   tags: workload-agent
+  
+- name: Find the exact path of the installed agent executable
```

Comment:
If coming from the rpm, the workload agent executable always gets installed into `/usr/bin/google_cloud_workload_agent`; I think we'd be fine using the default path, which will include `/usr/bin`.